pkg/exporter/sumologicexporter/README.md

```diff
@@ -159,6 +159,36 @@ Below is a list of all attribute keys that are being translated.
 
 ## Source Templates
 
+> **IMPORTANT NOTE**:
+>
+> When using non-`OTLP` based format (e.g. `JSON` for logs) metadata attributes
```

Comment:
```suggestion
> When using `HHTP` based format (e.g. `JSON` for logs) metadata attributes
```

Reply:
I wanted to use this wording but we're still using HTTP based OTLP right? (serialized OTLP payload, in HTTP request)

Reply:
:exploding_head:

Reply:
This IMPORTANT NOTE applies only to logs, doesn't it? Not to metrics or traces? I think this should be very explicitly stated here.

Great to see the examples! I'd go even further with them - add a sentence describing why the incorrect configuration will not work.

Reply:
This is for all of the pillars in case of using source headers

Only in logs they are send as separate header (`X-Sumo-FIelds`)